static/src/js/util/url/collectionsEncoders.js

```diff
@@ -193,6 +212,7 @@ const encodeRemovedGranules = (isOpenSearch, removedGranuleIds) => {
   return encodeGranules(isOpenSearch, removedGranuleIds)
 }
 
+// Decoders
```

Comment:
If you are using this to title a section, I would also title the encoders. Personally, I am not sure that the section title is necessary. 

If you do end up keeping it, I feel like the multiline `/** */` style of comment typically works better for a section title: